velox/dwio/parquet/tests/reader/ParquetReaderTest.cpp

```diff
@@ -1181,3 +1181,143 @@ TEST_F(ParquetReaderTest, testV2PageWithZeroMaxDefRep) {
   assertReadWithReaderAndExpected(
       outputRowType, *rowReader, expected, *leafPool_);
 }
+
+TEST_F(ParquetReaderTest, arrayOfMapOfIntKeyArrayValue) {
```

Comment:
Could you please also add the expected Velox type for this row in the comment? Same for all other new tests you added. Thanks!

Reply:
Added it as part of unit test `expectedVeloxType`